pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -66,9 +66,9 @@ type ResolvedPipelineTask struct {
 	// If the PipelineTask is a Custom Task, RunName and Run will be set.
 	CustomTask            bool
 	RunName               string
-	Run                   *v1alpha1.Run
+	Run                   v1beta1.RunObject
 	RunNames              []string
-	Runs                  []*v1alpha1.Run
```

Comment:
Renaming this to `RunObjects` will make it clearer that it's either Runs or CustomRuns